modules/monitoring-enabling-query-logging-for-thanos-querier.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+[id="enabling-query-logging-for-thanos-querier_{context}"]
+:_content-type: PROCEDURE
+= Enabling query logging for Thanos Querier 
+
+[role="_abstract"]
+For default platform monitoring in the `openshift-monitoring` project, you can enable the Cluster Monitoring Operator to log all queries run by Thanos Querier.
+
+[IMPORTANT]
+====
+Because log rotation is not supported, only enable this feature temporarily when you need to troubleshoot an issue. After you finish troubleshooting, disable query logging by reverting the changes you made to the `ConfigMap` object to enable the feature.
+====
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have created the `cluster-monitoring-config` `ConfigMap` object.
+
+.Procedure
+
+You can enable query logging for Thanos Querier in the `openshift-monitoring` project:
+
+. Edit the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
++
+. Add a `thanosQuerier` section under `data/config.yaml` and add values as shown in the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    thanosQuerier:
+      enableRequestLogging: <value> <1>
+      logLevel: <value> <2>
+      
+
+----
+<1> Set the value to `true` to enable logging and `false` to disable logging. The default value is `false`.
+<2> Set the value to `debug`, `info`, `warning`, or `error`. If no value exists for `logLevel`, the log level defaults to `error`.
```

Comment:
`warning` should be `warn`, see
https://github.com/openshift/openshift-docs/pull/44119#discussion_r843421544

Reply:
fixed